README.md

```diff
@@ -48,10 +48,11 @@ This repo contains the core NEAR Protocol node client.  It is written using the
 
 ```bash
 $ curl https://sh.rustup.rs -sSf | sh
+# you may need to restart your shell here
```

Comment:
we should add here:
```
source ~/.cargo/env
rustup default nightly
```

btw, what's wrong with . ~/.cargo/env?

Reply:
(e.g. restarting of the console doesn't work for me for example to get rustup/cargo working)

Reply:
> btw, what's wrong with . ~/.cargo/env?

Discussed with @david-stolp , our opinion that using `.` instead of `source` is an obscure bash feature that needs to be avoided in the documentation, at least because it confuses people.

Reply:
The console restart (or reloading `.bash_profile` on mac) was required when I was testing this on my machine to enable the `rustup` command.

Reply:
Restarting is needed even if you run manually source ~/.cargo/env?